CHANGELOG.md

```diff
@@ -19,7 +21,7 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [12.1.0] - 2018-08-28
 ### Added
-- new `check-efs.rb: checks cloudwatch metrics with the efs namespace for an arbitrary metric (@ivanfetch)
+- new `check-efs.rb`: checks cloudwatch metrics with the efs namespace for an arbitrary metric (@ivanfetch)
```

Comment:
you already made this change in another PR, please make this change in only one PR or it makes merging them more difficult.